layer.md

```diff
@@ -69,6 +69,8 @@ Where supported, MUST include file attributes for Additions and Modifications in
 - Non-directory files are considered "hardlinked" when their link count is greater than 1.
 - Hardlinked files are on a same device (i.e. comparing Major:Minor pair) and have the same inode.
 - The corresponding files that share the link with the > 1 linkcount may be outside the directory that the changeset is being produced from, in which case the `linkname` is not recorded in the changeset.
+- Union filesystem implementations may have limited or no support for hardlinks, particularly when a change is made to a hardlinked file or a hardlink is created to a file in a lower filesystem.
```

Comment:
Do you want to include this link? - https://docs.kernel.org/filesystems/overlayfs.html#index
Might be worth if when revisiting this section.

Reply:
Okay, I've included that.